src/helpers/map.js

```diff
@@ -64,6 +86,69 @@ export function isTraversable(tileID) {
 }
 
 
+/*
+ * Returns the radius, in pixels, of the given circular nontraversable tile ID.
+ *
+ * Circular nontraversable tiles include: powerups, bombs, active portals,
+ *   speed boosts, spikes, and buttons
+ */
+export function getNontraversableObjectRadius(tileID) {
```

Comment:
I had a hard time deciding whether or not to capitalize the 'T' there. I see now that our other nontraversable functions capitalize it. I'll fix it!